provisioning/roles/utils/celery/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Ensure supervisor is installed and up-to-date.
```

Comment:
Well, the problem is that factoring `utils/supervisor` into his proper role would lead to a single block which would ensure that supervisor is installed and that's all, I think?